doc/src/modules/physics/control/control_problems.rst

```diff
@@ -0,0 +1,341 @@
+Solved Textbook Problems using Control Module
+=============================================
+
+Given below, are some comprehensive textbook examples to demonstrate the possible use cases
+of the Control Module.
+
+Example 1
+---------
+
+        .. image:: Q1.svg
+
+A pole zero plot of an unknown **Transfer Function** is given above.
+
+1. Determine the exact Transfer Function if the continuous time **DC Gain** of the system is **20**.
+2. Is the TransferFunction **stable** or **unstable** in nature.
+3. Obtain the **unit impulse response** of the system.
+4. Find the initial value of the **time-domain response** of system without using the time domain equation.
+
+Solution-
+
+    >>> # Imports
+    >>> from sympy import symbols, I, limit, pprint, solve, oo
+    >>> from sympy.physics.control import TransferFunction
+
+    Subpart 1
+
+    >>> s,k = symbols('s k')
+    >>> a = k*(s+3)                     # Zero at -3 in S plane
+    >>> b = (s+1)*(s+2-I)*(s+2+I)       # Poles at -1, (-2,j) and (-2,-j) in S plane
+    >>> tf = TransferFunction(a,b,s)
+    >>> pprint(tf)
+               k*(s + 3)
+    -------------------------------
+    (s + 1)*(s + 2 - I)*(s + 2 + I)
+    >>> gain = tf.dc_gain()
+    >>> print(gain)
+    3*k*(2 - I)*(2 + I)/25
+    >>> K = solve(gain - 20,k)[0]               # Solve for k
+    >>> tf = tf.subs({k:K})                     # Reconstruct the TransferFunction using .subs()
+    >>> pprint(tf.expand())
+       100*s
+       ----- + 100
+         3
+    -------------------
+     3      2
+    s  + 5*s  + 9*s + 5
+
+    Subpart 2
+
+    >>> tf.is_stable()  # Expect True, since poles lie in the left half of S plane
+    True
+
+    Subpart 3
+
+    >>> from sympy import inverse_laplace_transform
+    >>> t = symbols('t',positive = True)
+    >>> # Convert from S to T domain for impulse response
+    >>> tf = tf.to_expr()
+    >>> Impulse_Response = inverse_laplace_transform(tf,s,t)
+    >>> pprint(Impulse_Response)
+          -t        -2*t
+     100*e     100*e    *cos(t)
+     ------- - ----------------
+        3             3
+
+    Subpart 4
+
+    >>> # Apply the Initial Value Theorem on Equation of S domain
+    >>> # limit(y(t), t, 0) = limit(s*Y(S), s, oo)
+    >>> limit(s*tf, s, oo)
+    0
+
+Example 2
+---------
+
+Find the Transfer Function of the following Spring-Mass dampering system : -
+
+        .. image:: Q2.svg
+
+
+Soltion -
+
+    >>> # Imports
+    >>> from sympy import Function, laplace_transform, laplace_initial_conds, laplace_correspondence, diff, Symbol, solve
+    >>> from sympy.abc import t,s
+    >>> from sympy.physics.control import TransferFunction
+    >>> y = Function('y')
+    >>> Y = Function('Y')
+    >>> u = Function('u')
+    >>> U = Function('U')
+    >>> k = Symbol('k') # Spring Constant
+    >>> c = Symbol('c') # Damper
+    >>> m = Symbol('m') # Mass of block
+
+        $The \ Diffential \ Equation \ of \ the \ system \ will \ be \ as \ follows:$
```

Comment:
This shouldn't be in the math mode and "Differential Equation" is not capitalized.